services/streams/streams/parsers.go

```diff
@@ -61,6 +65,17 @@ func (streamComp *StreamComponent) ToMap() map[string]any {
 	return streamCompMap
 }
 
+func (repetitive *Repetitive) ToMap() map[string]any {
+	repetitiveMap := make(map[string]any)
+	if repetitive.Value != nil {
+	    repetitiveMap["value"] = repetitive.Value
+	}
+	if repetitive.Path != nil {
```

Comment:
```suggestion
	if repetitive!= nil && repeetitive.Path!="" {
```
the default value of a string is an empty string and not nil